packages/http-client-js/src/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from "./components/index.js";
```

Comment:
@tadelesh if I understand your case correctly this is because your MCP emitter would refer our client components so we need to expose them.

Personally since our 3p emitter is not stable yet, I would prefer to expose it under experimental sub-path.

@joheredi how do you think?

Reply:
yes. it will be used in mcp http server emitter.